firebase-sessions/src/main/kotlin/com/google/firebase/sessions/settings/SessionsSettings.kt

```diff
@@ -28,6 +28,7 @@ import kotlin.time.Duration.Companion.minutes
 internal class SessionsSettings(val context: Context) {
 
   var localOverrideSettings = LocalOverrideSettings(context)
+  var remoteSettings = RemoteSettings(context)
```

Comment:
Should these be `private val` instead?

Reply:
Yes, made it private now. But eventually I think I will change it var since both of these would be a part of `SessionSettings` constructor.